pgcli/main.py

```diff
@@ -469,6 +469,9 @@ def cli(database, user, host, port, prompt_passwd, never_prompt, dbname,
         print('Version:', __version__)
         sys.exit(0)
 
+    if not os.path.exists(os.path.expanduser('~/.config/pgcli')):
+        os.mkdir(os.path.expanduser('~/.config/pgcli'))
```

Comment:
Nice job. I did not consider the situation that .config dir not exists. I have fixed it now. Thanks very much!